modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenPasswordGrantTestCase.java

```diff
@@ -159,10 +202,54 @@ public void atEnd() throws Exception {
         restClient.closeHttpClient();
         scim2RestClient.closeHttpClient();
         actionsRestClient.closeHttpClient();
+        client.close();
         accessToken = null;
         jwtClaims = null;
     }
 
+    @Test(groups = "wso2.is", description = "Get access token with client credentials grant")
+    public void testGetAccessTokenWithClientCredentialsGrant() throws Exception {
```

Comment:
This should be for password grant

Reply:
Fixed with the commit fde81a8b3f8a2f2928fb2efe23519d8d39da2833